pubspec.yaml

```diff
@@ -5,8 +5,8 @@ repository: https://github.com/AS-Devs/signalr_flutter
 issue_tracker: https://github.com/AS-Devs/signalr_flutter/issues
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.20.0"
+  flutter: ^3.19.0
+  sdk: ^3.2.0
```

Comment:
Do the same for SDK also. Do not constraint upper bound, constraint lower bound.